app/kumactl/pkg/config/io.go

```diff
@@ -10,8 +10,10 @@ import (
 	"github.com/pkg/errors"
 )
 
+var DefaultConfigFile = filepath.Join(os.Getenv("HOME"), ".kumactl", "config")
+
 func Load(file string, cfg *config_proto.Configuration) error {
-	configFile := defaultConfigFile()
+	configFile := DefaultConfigFile
```

Comment:
Sorry, can we change the way this logic gets integrated into the app flow ?

I would prefer to see an explicit `Save(DefaultConfiguration())` action, instead of having it hidden inside `Load`.

Let's add these instructions to `NewRootCmd()`, e.g.

```
if root.IsFirstTimeUsage() {
  if err := root.Save(DefaultConfiguration()); err != nil {
    return err
  }
}
return root.LoadConfig()
```